dojo/management/commands/dedupe.py

```diff
@@ -1,23 +1,27 @@
 from django.core.management.base import BaseCommand
 from pytz import timezone
 
-from dojo.models import Finding, Test_Type
-from dojo.utils import get_system_setting
+from dojo.models import Finding, Product
+from dojo.utils import calculate_grade, do_dedupe_finding, do_dedupe_finding_task, get_system_setting, mass_model_updater
 import logging
 
 locale = timezone(get_system_setting('time_zone'))
 
 logger = logging.getLogger(__name__)
 deduplicationLogger = logging.getLogger("dojo.specific-loggers.deduplication")
 
-"""
-Author: Aaron Weaver
-This script will update the hashcode and dedupe findings in DefectDojo:
-"""
+
+def generate_hash_code(f):
```

Comment:
Should we make it a method of object `Finding`?

Reply:
Also as we use this function only when defining a lambda, it's better to move the code of this one line function in the lambda definition directly